Quadriga/src/main/webapp/WEB-INF/views/public/navbar.jsp

```diff
@@ -1,10 +1,13 @@
 <li ${currentPage == "home" ? "class=\"active\"" : ""}><a
-        href="${pageContext.servletContext.contextPath}/sites/${project.unixName}">Home</a></li>
+	href="${pageContext.servletContext.contextPath}/sites/${project.unixName}">Home</a></li>
 <li ${currentPage == "networks" ? "class=\"active\"" : ""}><a
-        href="${pageContext.servletContext.contextPath}/sites/${project.unixName}/browsenetworks">Networks</a></li>
+	href="${pageContext.servletContext.contextPath}/sites/${project.unixName}/projectblog">Project
```

Comment:
you probably want to adjust the currentPage value here. the current page is not "networks", is it?